Updated the function withdraw() in the SimpleAuction and BlindAuction…

ConsentDevelopment opened the pull request:
… contracts to not use a 'throw' if a address send() fails.
# Documentation #Examples

ConsentDevelopment committed:
Updated the function withdraw() in the SimpleAuction and BlindAuction contracts to not use a 'throw' if a address send() fails.

#Documentation #Examples

graup commented:
Just chimin' in, I guess it would be good if you could format the code according to the [Style Guide](http://solidity.readthedocs.io/en/latest/style-guide.html) (4 spaces instead of tab, and curly brace on the same line as `if`).

ConsentDevelopment committed:
Updated the example changes to use 4 spaces for indentation and also updated
the opening braces to be on the same line as the declaration.

#Documentation #Examples

ConsentDevelopment committed:
Second pass -

Updated the example changes to use 4 spaces for indentation and also updated
the opening braces to be on the same line as the declaration.

#Documentation #Examples

ConsentDevelopment commented:
Thanks for the heads-up. I have committed new changes that remove the tabs and move the opening braces to the declarations.

ConsentDevelopment committed:
* Fixed a typo from the last commit and added line returns to make the functions more readable.

#Documentation #Examples
